jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/JibPluginConfigurationTest.java

```diff
@@ -40,18 +39,14 @@
 
   private static final MavenProject project = new MavenProject();
   @Mock private MavenSession session;
-  private Properties sessionProperties = new Properties();
+  private Properties sessionProperties;
   private JibPluginConfiguration testPluginConfiguration;
 
-  private void clearProperties() {
-    sessionProperties.clear();
-    project.getProperties().clear();
-  }
-
   @Before
   public void setup() {
+    project.getProperties().clear();
```

Comment:
not using a static project would also allow us to ignore cleaning up the state here.